js/repl/CodeMirrorPanel.tsx

```diff
@@ -30,7 +33,7 @@ export default function CodeMirrorPanel(props: Props) {
         <CodeMirror
           onChange={onChange}
           options={{
-            ...props.options,
```

Comment:
The `fileSize` option is not used by the codemirror component. So here we explicitly specify the passing-through options.